IDisposableAnalyzers/build/IDisposableAnalyzers.targets

```diff
@@ -0,0 +1,23 @@
+<Project>
+
+  <PropertyGroup>
+    <OwnershipAttributesPath Condition="'$(OwnershipAttributesPath)' == ''">$(MSBuildThisFileDirectory)OwnershipAttributes$(DefaultLanguageSourceExtension)</OwnershipAttributesPath>
+
+    <IncludeOwnershipAttributes Condition="'$(IncludeOwnershipAttributes)' == '' and Exists($(OwnershipAttributesPath))">true</IncludeOwnershipAttributes>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(IncludeOwnershipAttributes)' == 'true' and Exists($(OwnershipAttributesPath))">
+    <Compile Include="$(OwnershipAttributesPath)" Visible="false" />
+
+    <!-- Workaround for https://github.com/dotnet/wpf/issues/810 -->
+    <_GeneratedCodeFiles Include="$(OwnershipAttributesPath)" Visible="false" Condition="'$(UseWPF)' == 'true'" />
+  </ItemGroup>
+
+  <Target Name="OwnershipAttributesRequestedForUnsupportedLanguage"
+          Condition="'$(IncludeOwnershipAttributes)' == 'true' and !Exists($(OwnershipAttributesPath))"
+          BeforeTargets="Compile">
+
+    <Warning Code="IDISP0026" Text="The project specifies &lt;IncludeOwnershipAttributes&gt;true&lt;/IncludeOwnershipAttributes&gt;, but including attributes in this language is not yet supported." />
```

Comment:
At @jeremyVignelles's request, I changed the behavior to provide a custom warning.

Resulting behavior:

|       | No `<IncludeOwnershipAttributes>` in project | `true` in project                    | `false` in project |
|-------|----------------------------------------------|--------------------------------------|--------------------|
| C#    | ✔ Provides attributes*                       | ✔ Provides attributes                | 🚫 No attributes    |
| VB/F# | 🚫 No attributes*                             | ⚠ Build warning due to missing file* | 🚫 No attributes    |

Cells with <b>*</b> could be different if you want. For example, we could customize the build failure message or introduce warnings in the default scenario.

@JohanLarsson Should this take IDISP0026, or have no code?

Reply:
Should be `IDISP026`, I think next available is fine as it is just a serial with no special meaning.

Reply:
Updated.

Reply:
I think what this file should do is get fed into an c# -> VB converter to convert it to VB.

Also what about winforms, you guys talked about WPF but not much about winforms?

Reply:
Know what let me see if I can feed it into an c# -> VB/F# converter for the attribute it looks simple enough to do but how would the probs know if it is C#, VB, or F#?

Reply:
> Also what about winforms, you guys talked about WPF but not much about winforms?

Winforms doesn't do anything unusual. WPF generates a temporary csproj for .xaml and the ripple effects from that make it a special case.

> how would the probs know if it is C#, VB, or F#?

By using `$(DefaultLanguageSourceExtension)` which it does on line 4 of the .targets file.